File: NSDate/NSDate+Extension.m
//
//  NSDate+Extension.m
//  SmallVillage
//
//  Created by 李 on 16/6/22.
//  Copyright © 2016年 chenglv. All rights reserved.
//

#import "NSDate+Extension.h"

@implementation NSDate (Extension)

+(NSString *)getNowDate {
    
    NSDate *date = [NSDate date];
    NSDateFormatter *selfFormatter = [[NSDateFormatter alloc] init];
    [selfFormatter setDateFormat:@"yyyyMMddHHmmss"];
    NSString *str = [selfFormatter stringFromDate:date];
    return str;
}
+(NSString *)getTimestamp {
    NSDate *localDate = [NSDate date];
    NSString *timeSp = [NSString stringWithFormat:@"%ld", (long)[localDate timeIntervalSince1970]];
    return timeSp;
}


@end

File: Swift/NSDate+Extension.swift
//
//  NSDate.swift
//  SmallVillage
//
//  Created by 李 on 16/7/27.
//  Copyright © 2016年 chenglv. All rights reserved.
//

import Foundation

/// NSDateFormatter 常量
private let yl_sharedDateFormatter = NSDateFormatter()
/// 当前日历对象
private let yl_sharedCalendar = NSCalendar.currentCalendar()

/**
 在 iOS 开发中 NSDateFormatter / NSCalendar 性能异常糟糕！
 每次 `创建`／销毁非常消耗性能
 */
extension NSDate {
}
